iam-login-service/src/main/java/it/infn/mw/iam/api/scim/model/ScimUser.java

```diff
@@ -93,7 +94,8 @@ private ScimUser(@JsonProperty("id") String id, @JsonProperty("externalId") Stri
       @JsonProperty("userType") String userType,
       @JsonProperty("preferredLanguage") String preferredLanguage,
       @JsonProperty("locale") String locale, @JsonProperty("timezone") String timezone,
-      @JsonProperty("active") Boolean active, @JsonProperty("emails") List<ScimEmail> emails,
+      @JsonProperty("active") Boolean active, @JsonProperty("serviceAccount") Boolean serviceAccount,
```

Comment:
Unfortunately `serviceAccount` is not a valid SCIM User field. We need to move this inside ScimIndigoUser object which uses the SCIM extension mechanism

Reply:
https://www.rfc-editor.org/rfc/rfc7643.html#section-4.1

Reply:
Done